nanopub/client.py

```diff
@@ -72,13 +89,20 @@ def find_nanopubs_with_pattern(self, subj=None, pred=None, obj=None,
                             params=params,
                             max_num_results=max_num_results)
 
-    def find_things(self, type=None, searchterm=' ',
+    def find_things(self, type: str, searchterm=' ',
                     max_num_results=1000):
+        """Search things (experimental).
+
+        Search for any nanopublications that introduce a concept of the given type, that contain
+        text with the given search term.
+
+        Args:
+            type (str): A URI denoting the type of the introduced concept
+            searchterm (str): The term that you want to search on
+            max_num_results (int): Maximum number of result, default = 1000
+
         """
-        Searches the nanopub servers (at the specified grlc API) for any nanopubs of the given type, with given search term,
-        up to max_num_results.
-        """
-        if not type or not searchterm:
+        if not searchterm:
```

Comment:
Do we not need to check that type is also specified? If no longer true, then the error message should be clarified.

Reply:
Good point. I removed the default for `type` argument which makes that check redundant. But indeed the error message is now misleading. I updated it. We should ideally do more validation, but I think this is fine for now.